### PR TITLE
HPCC-26614 Document Helm Job Memory Sections

### PR DESCRIPTION
**Pull request by g-pan**
## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

## Testing:
Complete integration tested successfully
 http://10.240.61.86/view/Docs-gp/job/DocBuild-01/

### CODE REVIEW

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml [hunk 1]
```diff
@@ -361,6 +361,85 @@
         in bare metal we can have jobs that could effect other jobs because
         they are running in the same process space.</para>
       </sect3>
+
+      <sect3 id="YAML_Thor_and_hThor_Memory">
+        <title>Thor and hThor Memory </title>
+
+        <para>The "memory" sections were introduced initially for Thor only,
```
Comment (JamesDeFabia) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml:368]: Is this information necessary to share? I don't think anyone needs to know it was Thor only at first.

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml [hunk 2]
```diff
@@ -361,6 +361,85 @@
...
+        this example, <emphasis>workerMemory.query</emphasis> defines that 3G
+        of the available resourced memory should be assigned to query memory,
+        and 500M to "thirdParty" uses. </para>
+
+        <para>This limits the HPCC Systems memory
+        <emphasis>roxiemem</emphasis> usage to exactly 3G, leaving 1G free
+        other purposes. The "thirdParty" is not actually allocated, but is
+        used solely as part of the running total, to ensure that the
+        configuration doesn't specify a total in this section larger than the
+        resources section, e.g. if "thirdParty" was set to "2G" in the above
```
Comment (JamesDeFabia) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml:415]: e.g.,

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml [hunk 3]
```diff
@@ -361,6 +361,85 @@
...
+        section, there would be a runtime complaint when Thor ran that the
+        definition exceeded the resource limit. </para>
+
+        <para>It is also possible to override the default recommended
+        percentage (90% by default), by setting
+        <emphasis>maxMemPercentage</emphasis>. If "query" is not defined, then
+        it is calculated to be the recommended max memory minus the defined
+        memory (e.g. "thirdParty"). </para>
```
Comment (JamesDeFabia) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml:423]: e.g.,

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml [hunk 4]
```diff
@@ -361,6 +361,85 @@
...
+
+        <para>In Thor there are 3 resource areas,
+        <emphasis>eclAgent</emphasis>, <emphasis>ThorManager</emphasis>, and
+        <emphasis>ThorWorker</emphasis>(s). Each had a *Resource area that
+        defined their Kubernetes resource needs, and now they have a
```
Comment (JamesDeFabia) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml:428]: define, not defined

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
Just a few typos from my POV

**Review by JamesDeFabia — APPROVED** [review]
Looks ok from my POV, but Jake should be the final reviewer.

**Review by jakesmith — COMMENTED** [review]
A couple of minor change suggestions.

**Review by jakesmith — APPROVED** [review]
@g-pan - looks good, please squash

**Review by JamesDeFabia — APPROVED** [review]

**Pull request merged**